docs/_posts/2017-02-11-warts.md.dev

```diff
@@ -9,6 +9,14 @@ order: 1
 Here is a list of built-in warts under the
 `org.wartremover.warts` package.
 
+### ArrayEquals
+
+Unlike other collections `==` on arrays and iterators checks reference equality:
```

Comment:
I don't think it's particularly shocking that built-in equality of a mutable thing has the semantics of reference equality.  I do think it's surprising that two different mutable standard library collections types have totally different built-in equality semantics.  And I do think it risks bugs if you're using built-in equality on any sort of mutable collection, `Array`s included, but also `mutable.Seq`.

But perhaps I am out of touch with what people would expect from mutable collection.